applications/rag/README.md

```diff
@@ -56,9 +56,12 @@ Next, set up the inference server, the `pgvector` instance, Jupyterhub, Kuberay
 
 1. `cd ai-on-gke/applications/rag`
 
+2. (Optional) [Enable IAP and Set up Brand](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/modules/jupyter/authentication/README.MD#enable_iap_service) (Recommend)
```

Comment:
It's awkward to have this in between the `cd` command and the `tf apply`. Move this all to the bullet below and explain what every feature does. Even I am unfamiliar with these features, and our users will need information in order to decide:

* Optionally set `add_auth` to true, which will expose a public load balancer and allow you to log in with IAP using your Google credentials.
* Optionally [set up brand](...) (TODO: explain what this is and why the user would enable brand)